scripts/updatePubLocationAndLinks.py

```diff
@@ -28,17 +28,18 @@ def main():
             record.spatial = re.sub(charPattern, '', record.spatial)
 
             #Reversing download and reader values of webpub mainfest in has_part attribute 
-            hasPartManifest = record.has_part[0].split('|')
-
-            if hasPartManifest[len(hasPartManifest)-2] == 'application/webpub+json' \
-            and hasPartManifest[-1] == '{"catalog": false, "download": true, "reader": false, "embed": false}':
-                
-                hasPartManifest[-1] = '{"catalog": false, "download": false, "reader": true, "embed": false}'
-                newHasPart = []
-                newHasPart.append('|'.join(hasPartManifest))
-                for i in record.has_part[1:]:
-                    newHasPart.append(i)
-                record.has_part = newHasPart
+            if record.has_part:
```

Comment:
I sometimes like to invert to avoid more nesting:
```
if not record.has_part:
    continue
```
Also keeps the git history cleaner! But this seems fine here, just thought I'd mention!

Reply:
Sounds like a good organizational tip to remember. I'll add your implementation to the script.